pkg/image/registry_options.go

```diff
@@ -0,0 +1,16 @@
+package image
+
+// RegistryOptions for the OCI registry provider.
+type RegistryOptions struct {
+	InsecureSkipTLSVerify bool
+	Auth                  []RegistryCredentials
```

Comment:
Since this key is singular, it complicates following the code when it is used in a loop. Perhaps `AuthenticationMethods` or if the struct was called `Authentication` this could be `Methods`? Again, no strong suggestions, but as they are presented here, makes it hard to follow them when they are used

Reply:
agreed, I think we can do better than `Auth`. How about changing the field name from `Auth` to `Credentials`? That would solve both the plurality problem and more closely align with the struct name (`RegistryCredentials`) without being redundant on the word "Registry".

Reply:
done deal in the latest commit